### PR TITLE
Add sample integration tests

### PR DESCRIPTION
**Pull request by manoramahp**

### CODE REVIEW

File: modules/integration/src/test/java/org/apache/synapse/samples/framework/tests/transport/Sample266.java [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+            fail("Failed to load the Axis2BackEndServerController");
+        }
+
+        assertEquals(0, axis2Server.getMessageCount("SimpleStockQuoteService", "placeOrder"));
+
+        String addUrl = "tcp://localhost:6060/services/StockQuoteProxy";
+        StockQuoteSampleClient client = getStockQuoteClient();
+        client.placeOrder(addUrl, null, null, "IBM");
+
+        Thread.sleep(2000);
```
Comment (nuwanw) [inline modules/integration/src/test/java/org/apache/synapse/samples/framework/tests/transport/Sample266.java]: why sleep is needed ? can we use a iterator

File: modules/integration/src/test/java/org/apache/synapse/samples/framework/tests/transport/Sample267.java [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+            fail("Failed to load the Axis2BackEndServerController");
+        }
+
+        assertEquals(0, axis2Server.getMessageCount("SimpleStockQuoteService", "placeOrder"));
+
+        String addUrl = "udp://localhost:9999?contentType=text/xml";
+        StockQuoteSampleClient client = getStockQuoteClient();
+        client.placeOrder(addUrl, null, null, "IBM");
+
+        Thread.sleep(2000);
```
Comment (nuwanw) [inline modules/integration/src/test/java/org/apache/synapse/samples/framework/tests/transport/Sample267.java]: why sleep?

File: modules/integration/src/test/java/org/apache/synapse/samples/framework/tests/advanced/Sample657.java [hunk 3]
```diff
@@ -0,0 +1,48 @@
+/*
```
Comment (chanikag) [inline modules/integration/src/test/java/org/apache/synapse/samples/framework/tests/advanced/Sample657.java]: Year is missing in the license header?

File: modules/integration/src/test/java/org/apache/synapse/samples/framework/tests/message/Sample252.java [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+            fail("Failed to load the Axis2BackEndServerController");
+        }
+
+        assertEquals(0, axis2Server.getMessageCount("SimpleStockQuoteService", "placeOrder"));
+        JMSSampleClient client = new JMSSampleClient();
+        client.connect("dynamicQueues/JMSTextProxy");
+        client.sendTextMessage("12.33 1000 ACP");
+        Thread.sleep(2000);
+        assertEquals(1, axis2Server.getMessageCount("SimpleStockQuoteService", "placeOrder"));
+        client.shutdown();
```
Comment (chanikag) [inline modules/integration/src/test/java/org/apache/synapse/samples/framework/tests/message/Sample252.java]: Shutdown the client shouldn't be in a finally block? What happened if an exception occurred?

Comment (manoramahp) [reply modules/integration/src/test/java/org/apache/synapse/samples/framework/tests/message/Sample252.java]: Done

File: modules/integration/src/test/java/org/apache/synapse/samples/framework/tests/advanced/Sample657.java (hunk 3 above)
Comment (manoramahp) [reply modules/integration/src/test/java/org/apache/synapse/samples/framework/tests/advanced/Sample657.java]: Apache license header included in wso2-synapse files doesn't contain year

Comment (chanikag) [reply modules/integration/src/test/java/org/apache/synapse/samples/framework/tests/advanced/Sample657.java]: But some classes has. So then we might have to fix those.

File: modules/integration/src/test/java/org/apache/synapse/samples/framework/tests/transport/Sample266.java (hunk 1 above)
Comment (manoramahp) [reply modules/integration/src/test/java/org/apache/synapse/samples/framework/tests/transport/Sample266.java]: Replaced with waiting loops

File: modules/integration/src/test/java/org/apache/synapse/samples/framework/tests/transport/Sample267.java (hunk 2 above)
Comment (manoramahp) [reply modules/integration/src/test/java/org/apache/synapse/samples/framework/tests/transport/Sample267.java]: Replaced with waiting loops

File: modules/integration/src/test/java/org/apache/synapse/samples/framework/tests/advanced/Sample657.java (hunk 3 above)
Comment (riyafa) [reply modules/integration/src/test/java/org/apache/synapse/samples/framework/tests/advanced/Sample657.java]: I think the new files are licensed under wso2 and the forked files from apache still have the apache license header. Correct me if I am mistaken.

File: modules/integration/src/test/java/org/apache/synapse/samples/framework/tests/message/Sample381.java [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.synapse.samples.framework.tests.message;
+
+import org.apache.synapse.samples.framework.SynapseTestCase;
+
```
Comment (nuwanw) [inline modules/integration/src/test/java/org/apache/synapse/samples/framework/tests/message/Sample381.java]: java doc class comment is missing

File: modules/integration/src/test/java/org/apache/synapse/samples/framework/tests/advanced/Sample657.java (hunk 3 above)
Comment (manoramahp) [reply modules/integration/src/test/java/org/apache/synapse/samples/framework/tests/advanced/Sample657.java]: All the integration test classes have the same license header (doesn't contain year). So how are we going to proceed?

File: modules/integration/src/test/java/org/apache/synapse/samples/framework/tests/message/Sample381.java (hunk 5 above)
Comment (manoramahp) [reply modules/integration/src/test/java/org/apache/synapse/samples/framework/tests/message/Sample381.java]: Added

**Pull request merged**